psi4/src/read_options.cc

```diff
@@ -2626,10 +2626,13 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         /*- Convergence criterion for amplitudes (residuals). -*/
         options.add_double("R_CONVERGENCE", 1e-5);
         /*- Convergence criterion for RMS orbital gradient. Default adjusts
-        depending on |occ__e_convergence|. -*/
-        options.add_double("RMS_MOGRAD_CONVERGENCE", 1e-6);
-        /*- Convergence criterion for maximum orbital gradient -*/
-        options.add_double("MAX_MOGRAD_CONVERGENCE", 1e-3);
+        depending on |occ__e_convergence|. Default represents the default
```

Comment:
There is a lot of `default`'s in these strings. I don't really follow what which default means.